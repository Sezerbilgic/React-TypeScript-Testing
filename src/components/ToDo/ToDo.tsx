import React from "react";

const ToDo = () => {
  return (
    <div data-testid="todo-1">
      ToDoasdadasd
    </div>
  )
}

export default ToDo;