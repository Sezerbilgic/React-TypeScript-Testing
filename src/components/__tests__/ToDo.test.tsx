import { screen, render } from '@testing-library/react';
import renderer from "react-test-renderer";
import ToDo from "../ToDo/ToDo";

test('renders learn react link', () => {
  render(<ToDo />);
  const todoElement = screen.getByTestId('todo-1');
  expect(todoElement).toBeInTheDocument();
});


test('matches snaphot', () => {
  const tree = renderer.create(<ToDo />).toJSON();
  expect(tree).toMatchSnapshot();
})