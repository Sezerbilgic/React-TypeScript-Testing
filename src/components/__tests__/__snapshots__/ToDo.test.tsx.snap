// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snaphot 1`] = `
<div
  data-testid="todo-1"
>
  ToDoasdadasd
</div>
`;
